resources/idlharness.js

```diff
@@ -2684,7 +2685,7 @@ IdlInterface.prototype.test_members = function()
                 assert_false(member.name in this.get_interface_object().prototype,
                              "The prototype object must not have a property " +
                              format_value(member.name));
-            }.bind(this), this.name + " interface: member " + member.name);
+            }.bind(this), `${this.name} interface: ${member.type} ${desc}`);
```

Comment:
Can you make this test name consistent with the other test that checks the absence of properties?

Reply:
Done, please take another look, thanks.